hack/dockerized

```diff
@@ -46,6 +40,6 @@ _rsync --delete --exclude '.glide*' --exclude "cluster/vagrant/.kubeconfig" --ex
 docker run --rm -v "${BUILDER}:/root:rw,z" -w "/root/go/src/kubevirt.io/kubevirt" -it ${BUILDER} "$@"
 
 # Copy the whole kubevirt data out to get generated sources and formatting changes
-_rsync --exclude '.glide*' --exclude "vendor" --exclude "_out" --exclude ".vagrant" --exclude ".git" "rsync://root@127.0.0.1:${RSYNCD_PORT}/build" ${KUBEVIRT_DIR}/
+_rsync --exclude '.glide*' --exclude "_out" --exclude ".vagrant" --exclude ".git" "rsync://root@127.0.0.1:${RSYNCD_PORT}/build" ${KUBEVIRT_DIR}/
```

Comment:
Genuinely curious, why do we no longer exclude vendor when copying build out?

Reply:
It is important if you are also looking into the vendor dependencies. Another option is to commit all vendor dependencies into the source.

Reply:
When that comes up, consider this forward +1. Github even hides vendor/ diffs.